client/routes/index.jsx

```diff
@@ -60,6 +66,8 @@ const GeneralRoutes = () => {
             </Menu>
 
             <Route exact path="/" component={ScenariosList} />
+            <Route exact path="/official" component={ScenariosList} />
+            <Route exact path="/community" component={ScenariosList} />
```

Comment:
Do we think we'll want to support more than two categories? If so, it might be more extendable to add filter checkboxes to the scenario listing page rather than a lot of items to the main menu.

Reply:
@ktbee I told @evmiguel to make these menu additions for the purpose of easy access to the feature for demonstration. The next iteration should put them all in a `<Dropdown.Menu>`

Reply:
Sounds good!